docs/reference.md

```diff
@@ -635,13 +702,17 @@ Methods used to provide payment during a successful checkout flow:
 
 ## Errors
 
-Error                                               | Cause                                                               | Returned by
---------------------------------------------------- | ------------------------------------------------------------------- | ---
-<a id="e1">`USAGE_ERROR`</a>                        | Reader SDK was used in an unexpected or unsupported way.            | all methods
-<a id="e2">`AUTHORIZE_NO_NETWORK`</a>               | Reader SDK could not connect to the network.                        | [authorizeAsync](#authorizeasync)
-<a id="e3">`CHECKOUT_CANCELED`</a>                  | The user canceled the checkout flow.                                | [startCheckoutAsync](#startcheckoutasync)
-<a id="e4">`CHECKOUT_SDK_NOT_AUTHORIZED`</a>        | The checkout flow started but Reader SDK was not authorized.        | [startCheckoutAsync](#startcheckoutasync)
-<a id="e5">`READER_SETTINGS_SDK_NOT_AUTHORIZED`</a> | The Reader settings flow started but Reader SDK was not authorized. | [startReaderSettingsAsync](#startreadersettingsasync)
+Error                                                    | Cause                                                               | Returned by
+-------------------------------------------------------- | ------------------------------------------------------------------- | ---
+<a id="e1">`USAGE_ERROR`</a>                             | Reader SDK was used in an unexpected or unsupported way.            | all methods
+<a id="e2">`AUTHORIZE_NO_NETWORK`</a>                    | Reader SDK could not connect to the network.                        | [authorizeAsync](#authorizeasync)
+<a id="e3">`CHECKOUT_CANCELED`</a>                       | The user canceled the checkout flow.                                | [startCheckoutAsync](#startcheckoutasync)
+<a id="e4">`CHECKOUT_SDK_NOT_AUTHORIZED`</a>             | Reader SDK was not authorized.                                      | [startCheckoutAsync](#startcheckoutasync)
```

Comment:
"Reader SDK was not authorized" is used as the cause for multiple errors. We should differentiate based on the situation in which the error is encountered. This is why the original text included "The checkout flow started but".